ironfish/src/metrics/meter.ts

```diff
@@ -32,7 +32,7 @@ export class Meter {
   private _intervalLastMs: number | null = null
 
   constructor(options?: { maxRollingAverageSamples?: number }) {
-    this._intervalMs = 1000
+    this._intervalMs = 1
```

Comment:
Prob should not change this

Reply:
Yeah I can pass in as a param. 
The reason I change the interval is because the rate is almost nearly zero during the 1000 ms window. The rate is more accurate for shorter interval.